src/dotnet/APIView/APIViewWeb/Controllers/AutoReviewController.cs

```diff
@@ -18,6 +18,7 @@
 using Microsoft.Extensions.Logging;
 using Microsoft.TeamFoundation.SourceControl.WebApi;
 using Microsoft.VisualStudio.Services.Account;
+using Octokit;
```

Comment:
nit: unused

Reply:
@chidozieononiwu could you make sure to clean this in your next PR?